PusherMainView/PusherMainView/AuthTokenViewcontroller.swift

```diff
@@ -10,6 +10,15 @@ final class AuthTokenViewcontroller: NSViewController {
     
     required init?(coder: NSCoder) {
         super.init(coder: coder)
+        #if DEBUG
+        print("\(self.className) init")
+        #endif
```

Comment:
remove?

Reply:
I let it in purpose so we can see init and deinit calls in console.